src/ui/components/keyboard_switch.h

```diff
@@ -18,7 +18,7 @@
 #include <screen.h>
 #include <ui/component.h>
 
-typedef enum { DIGITS, LOWER_CASE, UPPER_CASE, NUM_INPUT_TYPES } keyboard_mode_t;
+typedef enum { DIGITS, LOWER_CASE, UPPER_CASE } keyboard_mode_t;
```

Comment:
`NUM_INPUT_TYPES` can be used as a tracker for the size of the enum, without having to hardcode it in the code or change it if additional values are added to the enum itself (see below)

Reply:
Yeah I was aware; see the commit message / my comment below.